packages/esm-patient-chart-app/src/actions-buttons/add-past-visit.test.tsx

```diff
@@ -2,8 +2,15 @@ import React from 'react';
 import AddPastVisitOverflowMenuItem from './add-past-visit.component';
 import { screen, render } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
+import { showModal } from '@openmrs/esm-framework';
+
+const mockShowModal = showModal as jest.Mock;
+
+jest.mock('@openmrs/esm-framework', () => {
+  const originalModule = jest.requireActual('@openmrs/esm-framework');
+  return { originalModule, showModal: jest.fn() };
```

Comment:
This is missing from the framework mocks?

Reply:
Added the function to framework mocks [#403](https://github.com/openmrs/openmrs-esm-core/pull/403)